components/backup/tests/integrations/test_backup.rs

```diff
@@ -234,7 +234,7 @@ fn test_backup_and_import() {
             // Prewrite
             let start_ts = suite.alloc_ts();
             let mut mutation = Mutation::default();
-            mutation.op = Op::Put;
+            mutation.set_op(Op::Put);
```

Comment:
why do we need to change this?

Reply:
Because Prost and rust-proto encode enums in different ways, other uses have already been changed